src/NServiceBus.Core.Tests/Transports/MulticastTransportOperationTest.cs

```diff
@@ -14,7 +15,7 @@ public void Should_not_share_constraints_when_not_provided()
             var transportOperation = new MulticastTransportOperation(new OutgoingMessage(Guid.NewGuid().ToString(), new Dictionary<string, string>(), new byte[0]), typeof(Guid));
             var secondTransportOperation = new MulticastTransportOperation(new OutgoingMessage(Guid.NewGuid().ToString(), new Dictionary<string, string>(), new byte[0]), typeof(Guid));
 
-            transportOperation.DeliveryConstraints.Add(new NonDurableDelivery());
+            transportOperation.DeliveryConstraints.Add(new DiscardIfNotReceivedBefore(TimeSpan.FromDays(1)));
```

Comment:
Why replaced with a TTBR? I can't get that info from the test class/method names.

Reply:
I just picked one of the ones we still have around since it doesn't matter for the tests?